File: SOLID/Java/LSP/CloudUploader.java
class CloudUploader implements FileUploader {
    private static final long MAX_SIZE = 100;

    public boolean canUpload(long sizeInMb) {
        return sizeInMb <= MAX_SIZE;
    }

    public void upload(String fileName, long sizeInMb) {
        System.out.println("Cloud upload: " + fileName);
    }
}

File: SOLID/TS/LSP/CloudUploader.ts
import { FileUploader } from "./FileUploader";
export class CloudUploader implements FileUploader {
    private readonly MAX_SIZE = 100;

    canUpload(sizeInMb: number): boolean {
        return sizeInMb <= this.MAX_SIZE;
    }

    upload(fileName: string, sizeInMb: number): void {
        console.log(`Cloud upload: ${fileName}`);
    }
}